src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/pool-list/pool-list.component.html

```diff
@@ -1,33 +1,51 @@
-<cd-table [data]="data"
-          columnMode="flex"
-          [columns]="columns"
-          identifier="name"
-          forceIdentifier="true"
-          [autoReload]="-1"
-          (fetchData)="refresh()"
-          selectionType="single"
-          (updateSelection)="updateSelection($event)"
-          [status]="tableStatus">
-  <cd-table-actions class="table-actions"
-                    [permission]="permission"
-                    [selection]="selection"
-                    [tableActions]="tableActions">
+<cd-table
+  #table
+  [data]="data"
+  columnMode="flex"
+  [columns]="columns"
+  identifier="name"
+  [searchableObjects]="true"
+  [serverSide]="true"
+  forceIdentifier="true"
+  selectionType="single"
+  [hasDetails]="true"
+  [status]="tableStatus"
+  [maxLimit]="25"
+  [autoReload]="-1"
+  (setExpandedRow)="setExpandedRow($event)"
+  (updateSelection)="updateSelection($event)"
+>
+  <cd-table-actions
+    class="table-actions"
+    [permission]="permission"
+    [selection]="selection"
+    [tableActions]="tableActions"
+  >
   </cd-table-actions>
+
+  <cd-pool-peers *cdTableDetail
+                  [selection]="expandedRow"
+                  [poolName]="poolName">
+  </cd-pool-peers>
 </cd-table>
 
 <ng-template #healthTmpl
              let-row="data.row"
              let-value="data.value">
-  <span [ngClass]="row.health_color | mirrorHealthColor">{{ value }}</span>
+  <span [ngClass]="row.health_color | mirrorHealthColor">
+    {{ value }}
+  </span>
 </ng-template>
 <ng-template #localTmpl>
   <span i18n
         i18n-ngbTooltip
-        ngbTooltip="Local image count"># Local</span>
+        ngbTooltip="Local image count"># Local
+  </span>
 </ng-template>
 <ng-template #remoteTmpl>
   <span i18n
         i18n-ngbTooltip
-        ngbTooltip="Remote image count"># Remote</span>
+        ngbTooltip="Remote image count"># Remote
+  </span>
```

Comment:
Same thing here, this styling changes are not needed, right?

Reply:
which styling changes @Pegonzal . I guess we are applying color to the element on the basis of health